cosmic_ray/commands/new_config.py

```diff
@@ -24,21 +24,19 @@ def new_config():
 
     Returns: A new configuration as a single string.
     """
-    conf = {}
-
-    conf['module'] = qprompt.ask_str("Top-level module")
+    conf = {'module': qprompt.ask_str("Top-level module")}
 
     menu = qprompt.Menu()
-    for i, r in enumerate(cosmic_ray.plugins.test_runner_names()):
-        menu.add(str(i), r)
+    for at_pos, test_name in enumerate(cosmic_ray.plugins.test_runner_names()):
```

Comment:
The name `test_name` is incorrect. It should be something like `test_runner` or `test_runner_name` (or just `r`).